todo-app/src/Time.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+export default class Time extends React.Component{
+    
+    render(){
+        return (
+
+            <div>
+                {this.props.title}
+                {this.props.time}
```

Comment:
It would be nice if the `Time` component could format the time properly. Currently, the date is shown as it was entered, but perhaps you can use `momentjs` to show it nicer?